ISISScientificComputing/autoreduce
# Rename all queues to be pythonic and less confusing

# Changes
Note this is a very significant change and needs to be thoroughly tested to ensure it does not break anything.

* Renamed `AutoreductionProcessor` -> `reduction_proecessor` - This is because this processor merely is the interface between the queue and the reduction step as such the `auto` part of the word should not be used as it is not automatic.
* Renamed `QueueProcessor` -> `flow_processor` - This has been renamed to `flow` as it is a processor designed to deal with project flow between queues e.g. `dataready` -> `reductionpending` -> `processing` -> `etc.`

# Test
* Ensure that the queue_processors are working. Best to do this with the manual submission scripts on the development env.
* Ensure the pylint issues have been resolved (see travis output)

**Note: I will update the score once the other - less volatile PRs have gone through. As such I will leave the in progress label on this for now**

Fixes #123
